SCClassLibrary/Common/Files/SoundFile.sc

```diff
@@ -420,24 +420,23 @@ SoundFile {
 						Out.ar(out, sig * env * amp)
 					}).add;
 					~instrument = defname;
-					condition = Condition.new;
-					server.sync(condition);
+					server.sync;
 				};
 				ev.synth;	// set up as a synth event (see Event)
 				~bufnum =  server.bufferAllocator.alloc(1);
-				~bufferSize = 0x10000;
+				~bufferSize = ~bufferSize ? 0x10000;
 				~firstFrame = ~firstFrame ? 0;
 				~lastFrame = ~lastFrame ? numFrames;
-				~sustain = (~lastFrame - ~firstFrame)/(sampleRate ?? {server.options.sampleRate ? 44100});
+				~sustainTime = (~lastFrame - ~firstFrame)/(sampleRate ?? {server.sampleRate ? 44100});
```

Comment:
why `sustainTime` ? `sustain` is the standard value for the length of a sound event.

Reply:
ok, makes sense.